valve-monitor/web-app/src/adapters/nextjs/NextJsUrlManager.ts

```diff
@@ -5,6 +5,7 @@ export const NextJsUrlManager = {
     `${notehubUrl}/project/${projectUID}`,
 
   getValveMonitorDeviceData: () => `/api/valve-device-monitors`,
+  deviceNameUpdate: (deviceUID: string) => `/api/device/${deviceUID}/name`,
```

Comment:
I'd recommend having the `valve-monitor-config` device API file handle all device level updates instead of multiple files. I'm pretty sure it has most of the logic to handle name updates already